survey/application/src/main/java/me/nalab/survey/application/service/findfeedback/FeedbackFindService.java

```diff
@@ -36,8 +44,28 @@ public List<FeedbackDto> findAllFeedbackDtoBySurveyId(Long surveyId) {
 		return feedbackList.stream().map(FeedbackDtoMapper::toDto).collect(Collectors.toList());
 	}
 
+	@Transactional
+	@Override
+	public void updateFormFeedbackEntityIsReadBySurveyId(Long surveyId) {
+		List<Feedback> feedbacks = feedbackFindPort.findAllFeedbackBySurveyId(surveyId);
+
+		if (feedbacks.isEmpty()) return;
+
+		List<List<FormQuestionFeedbackable>> listList = feedbacks.stream()
```

Comment:
listList가 무엇인지 파악하기 힘든거 같아서 이렇게 바꾸는건 어떨까요

```suggestion
		List<List<FormQuestionFeedbackable>> formQuestionFeedbackList = feedbacks.stream()
```